README.rst

```diff
@@ -28,12 +28,22 @@ How can I contribute?
 What do I need to contribute?
 -----------------------------
 
-* A host with python 3.6+, gcc and the python development header files
-* tox
-* go
+* A host with `python` 3.6+ and `tox`
 * docker and/or podman+buildah
 * vagrant (optional, can be used to test FIPS mode and registered hosts)
 
+Test setup on openSUSE
+^^^^^^^^^^^^^^^^^^^^^^
+
+To setup the test environment on openSUSE Leap 15.6 and Tumbleweed, run the following commands:
+
+
+.. code-block:: shell-session
+
+    $ sudo zypper -n in podman buildah docker git-core python3 python3-pip
+    $ pip3 --quiet install --upgrade pip
```

Comment:
why not simply install python3-pip (or python311-pip) instead? seems simpler than this process imho.

Reply:
But python3-pip is installed just in the line before. This is there to update pip, which we do, but I'm not sure if that's really needed.

Reply:
It's not needed.

Reply:
This also needs to work on older versions, so this is the most generic version that will work at least on Leap and Tumbleweed.

If we advance to specifying python311 it will not work on Leap and require us to update the documentation every time there is a Leap release or Tumbleweed updates the major python version in use.

For the sake of making the document age better, I would stay with the generic version.